otlp/common.go

```diff
@@ -233,6 +234,22 @@ func getDataset(ri RequestInfo, attrs map[string]interface{}) string {
 	return dataset
 }
 
+func getLogsDataset(ri RequestInfo, attrs map[string]interface{}) string {
+	var dataset string
+	serviceName, ok := attrs["service.name"].(string)
+	if !ok || strings.TrimSpace(serviceName) == "" || strings.HasPrefix(serviceName, "unknown_service") {
+		if strings.TrimSpace(ri.Dataset) == "" {
+			// In practice, this won't actually get hit, because we error on missing dataset header
```

Comment:
I think we're switching to not erroring if there is no dataset, so this might get hit? Or do I have things backwards? 🙈

Reply:
Nah, you got it right. Should be resolved!